components/brave_wallet/renderer/eth_request_parser.cc

```diff
@@ -0,0 +1,122 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/renderer/eth_request_parser.h"
+
+#include <utility>
+#include <vector>
+
+#include "base/json/json_reader.h"
+#include "base/logging.h"
+#include "base/strings/string_number_conversions.h"
+#include "brave/components/brave_wallet/common/web3_provider_constants.h"
+
+namespace brave_wallet {
+
+absl::optional<base::Value> GetObjectFromParamsList(const std::string& json) {
```

Comment:
put this in an anonymous namespace?

Reply:
Fixed in d8c9c09e62fadd786f09bfdf932a2573f3dd5358